docker/docker-compose.yaml

```
@@ -65,3 +65,14 @@ services:
       - ./log:/var/log
     ports:
       - "1326:1326"
+  vulsrepo:
+      image: vuls/vulsrepo:latest
+      command:
+        - vulsrepo-server
+      volumes:
+        - ./log:/var/log
+        - ./vulsresults:/vuls/results
```

To make sure we pick up the volume data on the server.

```suggestion
      volumes:
        - ./log:/var/log
        - ./vuls:/vuls
```